spec/controllers/replies_controller_spec.rb

```diff
@@ -70,12 +70,32 @@
     end
 
     it "requires post read if no unread_warned param" do
-      user_id = login
       reply_post = create(:post)
+      login_as(reply_post.user)
+      reply_post.mark_read(reply_post.user)
+      create(:reply, post: reply_post)
 
-      post :create, reply: {post_id: reply_post.id, user_id: user_id}
+      post :create, reply: {post_id: reply_post.id, user_id: reply_post.user_id}
       expect(response.status).to eq(200)
-      expect(flash[:error]).to eq("There are unread replies not shown here. (Click 'post' again if you wish to ignore this warning.)")
+      expect(flash[:error]).to eq("There have been 1 new replies since you last viewed this post.")
```

Comment:
I'd want to pluralize this properly in the controller – "There has been 1 new reply"?